homeassistant/components/fritzbox_callmonitor/config_flow.py

```diff
@@ -0,0 +1,130 @@
+"""Config flow for fritzbox_callmonitor."""
+
+from fritzconnection.core.exceptions import FritzConnectionException
+import requests
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_PORT, CONF_USERNAME
+
+from .base import FritzBoxPhonebook
+
+# pylint:disable=unused-import
+from .const import (
+    CONF_PHONEBOOK,
+    CONF_PREFIXES,
+    DEFAULT_HOST,
+    DEFAULT_PHONEBOOK,
+    DEFAULT_PORT,
+    DEFAULT_USERNAME,
+    DOMAIN,
+)
+
+DATA_SCHEMA_USER = vol.Schema(
+    {
+        vol.Required(CONF_HOST, default=DEFAULT_HOST): str,
+        vol.Required(CONF_PORT, default=DEFAULT_PORT): vol.Coerce(int),
+        vol.Required(CONF_USERNAME, default=DEFAULT_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Optional(CONF_PHONEBOOK, default=DEFAULT_PHONEBOOK): int,
+        vol.Optional(CONF_PREFIXES): str,
```

Comment:
Should we show this in the config flow ? Or is this an advanced feature that we can move to the options ?

Reply:
Now that you mention advanced features I would say this is more fitting for an option. Will add option support to this config flow for this advanced feature.

Reply:
Done in 99a2bb18b79788050ff028f8d4336183a98f40ff